blob/src/main/java/io/crate/blob/v2/BlobIndicesService.java

```diff
@@ -74,7 +72,7 @@ public BlobIndicesService(Settings settings,
         super(settings);
         this.clusterService = clusterService;
         this.indicesLifecycle = indicesLifecycle;
-        this.listener = new LifecycleListener();
+        indicesLifecycle.addListener(new LifecycleListener());
```

Comment:
We've had issues before using `addListener` in the constructor (see 4f68d8b4c6e87a051e288ee437c9f6f3700a8c9f). Isn't that a problem here as well?